src/Pagination/Pagination.js

```diff
@@ -89,7 +89,7 @@ export class Pagination extends Component {
         // calculate number of pages based on item total and items per page
         // check to make sure itemsPerPage != 0
         this.numberOfPages = Math.ceil(
-            itemsTotal / (itemsPerPage ? itemsPerPage : 10)
+            itemsTotal / (itemsPerPage || 10)
```

Comment:
With a default value, we could remove the `|| 10`.

Reply:
I thought about this, but if the consumer passes in a `0` then this will break

Reply:
Fair enough.  If it would break with certain values, then we should consider adding a custom prop type validator that would prevent that.  But, that would be a follow-up enhancement if we wanted to do that.